v2/internal/controllers/user_assigned_identities_test.go

```diff
@@ -111,3 +113,26 @@ func Test_MySQLServerWithKubernetesReferenceUserAssignedIdentity(t *testing.T) {
 		tc.Expect(armID).To(testcommon.EqualsIgnoreCase(*mi.Status.Id))
 	}
 }
+
+func createPasswordSecret(
```

Comment:
Why not move this test?

Reply:
I moved all the files that adhered to our naming convention (ie `crd_*.go`); can move other files if neede, but I'd like to do that in a later PR.

Reply:
I think every test in this folder should move. This folder isn't _just_ CRD tests it's all end to end tests and I think they should be together regardless of if they are for CRUD for a single CRD, testing edge cases, testing other features (configmap export, etc).

To that effect a better folder name might be `endtoendtests` but I'm fine w/ crdtests too.

Reply:
Otherwise if we don't move (as @super-harsh pointed out) it makes the taskfile more complicated because the envtest tests are now split across directories and we need to run both. It removes some of the parallelism we've added within an envtest run, because if split across directories two tests that share the same envtest configuration will not run on the same envtest instance but instead 2 instances.

In fact, they won't run on 2 instances in parallel today unless we modify the taskfile to launch both `go test` commands in parallel, which IIRC we don't currently do for envtest runs to avoid noisy neighbor effects.

tl;dr: IMO we should move every test in this folder or none of them, not partially. They are all the same type of test (end to end). Some are CRD CRUD and some are other types of end to end tests but they belong together.